djstripe/models.py

```diff
@@ -106,7 +124,7 @@ def __str__(self):
 @python_2_unicode_compatible
 class Event(StripeObject):
     stripe_api_name = "Event"
-
+    objects = models.Manager()
```

Comment:
Sorry I was pretty exhausted when I wrote that comment. I didn't notice that you used ``stripe_objects`` to hold the stripe manager instead of just ``objects``. Totally fine with me and is definitely the way to go.

You can take out all of the ``objects = models.Manager()`` lines though, since that's already default.